src/QuartzNET-DynamoDB.Tests/Integration/Repository/RepositoryTests.cs

```diff
@@ -51,6 +51,110 @@ public void PersistTwoSchedulersSameId_OneRecord()
             Assert.Equal(initialSchedulerCount + 1, finalCount);
         }
 
+        /// <summary>
+        /// Tests the repository Store method that wraps the Dynamo BatchWriteItem API.
+        /// </summary>
+        [Fact]
+        [Trait("Category", "Integration")]
+        public void StoreMultipleEntities()
+        {
+            _testFactory = new DynamoClientFactory();
+            var client = _testFactory.BootStrapDynamo();
+            _sut = new Repository<DynamoScheduler>(client);
+
+            int initialSchedulerCount = _sut.Scan(null, null, null).Count();
+
+            var scheduler1 = new DynamoScheduler
+            {
+                InstanceId = "testInstance" + DateTime.UtcNow.Ticks.ToString(),
+                ExpiresUtc = (SystemTime.Now() + new TimeSpan(0, 10, 0)).UtcDateTime,
+                State = "Running"
+            };
+
+            var scheduler2 = new DynamoScheduler
+            {
+                InstanceId = "testInstance2" + DateTime.UtcNow.Ticks.ToString(),
+                ExpiresUtc = (SystemTime.Now() + new TimeSpan(0, 10, 0)).UtcDateTime,
+                State = "Running"
+            };
+
+            _sut.Store(new List<DynamoScheduler> {scheduler1, scheduler2});
+
+            int finalCount = _sut.Scan(null, null, null).Count();
+
+            Assert.Equal(initialSchedulerCount + 2, finalCount);
+        }
+
+        /// <summary>
+        /// Tests the repository Store method that wraps the Dynamo BatchWriteItem API.
+        /// 25 is the maximum number of items for one batch.
+        /// </summary>
+        [Fact]
+        [Trait("Category", "Integration")]
+        public void StoreTwentyFiveMultipleEntities()
```

Comment:
should this be StoreTwentyFiveEntities?

given this test tests the store method for a single batch, do we need the StoreMultipleEntities() test?